influx.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	influxClient "github.com/GeertJohan/influxdb-client-old"
```

Comment:
last time I checked, influxdb was still using 0.8 for their hosting+grafana service. Not sure if its been upgraded since